feature.js

```diff
@@ -0,0 +1,108 @@
+  img: 'img/CZ-binance.jpg',
+};
+
+const lastSpk = {
+  name: 'Brian Armstrong',
+  position: 'CEO of Coinbase.',
+  role: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Magnam fuga rem sequi dolore',
+  img: 'img/brian-armstrong-coinbase.jpg',
+};
+
+const spkArr = [firstSpk, secondSpk, thirdSpk, fourthSpk, fifthSpk, lastSpk];
+
+window.addEventListener('load', () => {
+  const fspCont = document.querySelector('#fsp');
+
+  fspCont.innerHTML += `<h3 class="feat">Featured speakers</h3>
+    <div class="sub-cont">
+       <div class="hold">
+            <div class="speaker">
+               <img class="spk-img" src="${spkArr[0].img}" alt="speaker one">
+               <div class="spk-text">
+                  <h3 class="name">${spkArr[0].name}</h3>
+                  <i class="position">${spkArr[0].position}</i>
+                  <p class="role">${spkArr[0].role}</p>
+               </div>
+            </div>
+            <div class="speaker">
+               <img class="spk-img" src="${spkArr[1].img}" alt="speaker two">
+               <div class="spk-text">
+                  <h3 class="name">${spkArr[1].name}</h3>
+                  <i class="position">${spkArr[1].position}</i>
+                  <p class="role">${spkArr[1].role}</p>
+               </div>
+            </div>
+       </div>
+       <span id="break"></span>
+       <div class="hold activ">
+            <div class="speaker">
+               <img class="spk-img" src="${spkArr[2].img}" alt="speaker three">
+               <div class="spk-text">
+                  <h3 class="name">${spkArr[2].name}</h3>
+                  <i class="position">${spkArr[2].position}</i>
+                  <p class="role">${spkArr[2].role}</p>
+               </div>
+            </div>
+            <div class="speaker">
+               <img class="spk-img" src="${spkArr[3].img}" alt="speaker four">
+               <div class="spk-text">
+                  <h3 class="name">${spkArr[3].name}</h3>
+                  <i class="position">${spkArr[3].position}</i>
+                  <p class="role">${spkArr[3].role}</p>
+               </div>
+            </div>
+       </div>
+
+       <div class="hold activ">
+            <div class="speaker">
+               <img class="spk-img" src="${spkArr[4].img}" alt="speaker five">
+               <div class="spk-text">
+                  <h3 class="name">${spkArr[4].name}</h3>
+                  <i class="position">${spkArr[4].position}</i>
+                  <p class="role">${spkArr[4].role}</p>
+               </div>
+            </div>
+            <div class="speaker">
+               <img class="spk-img" src="${spkArr[5].img}" alt="speaker six">
+               <div class="spk-text">
+                  <h3 class="name">${spkArr[5].name}</h3>
+                  <i class="position">${spkArr[5].position}</i>
+                  <p class="role">${spkArr[5].role}</p>
+               </div>
+            </div>
+       </div>
+   </div>
+    `;
+});
```

Comment:
- [ ] You have done an amazing job with trying to generate the speaker's section. However, as the [previous reviewer](https://github.com/elfin-git/Capstone-project-1/pull/4#discussion_r931142502) mentioned above and also according to the project requirements (screenshot below) you are required to generate this section dynamically. Currently, this is not the case. Kindly fix this.

![image](https://user-images.githubusercontent.com/86318284/181300029-07dd53f4-9ea2-4336-a203-317ad8792c16.png)

As you have everything already set up, I had written starter code for another student showing him a very basic example of how to generate the sections dynamically which you can have a look at [here](https://github.com/rebaz36/Portfolio/pull/12#discussion_r925688836). Kindly make this change and your project should be ready for approval.